python_modules/dagster/dagster/_core/definitions/freshness_checks/shared_builder.py

```diff
@@ -42,7 +41,7 @@ def build_freshness_multi_check(
     deadline_cron: Optional[str],
     timezone: str,
     severity: AssetCheckSeverity,
-    lower_bound_delta: datetime.timedelta,
+    lower_bound_delta: Optional[datetime.timedelta],
```

Comment:
is it valid for this and deadline_cron to be None at the same time?

Reply:
nope